homeassistant/components/august/sensor.py

```diff
@@ -231,10 +230,15 @@ async def async_added_to_hass(self) -> None:
         await super().async_added_to_hass()
 
         last_state = await self.async_get_last_state()
-        if not last_state or last_state.state == STATE_UNAVAILABLE:
+        last_sensor_state = await self.async_get_last_sensor_data()
+        if (
+            not last_state
+            or not last_sensor_state
+            or last_state.state == STATE_UNAVAILABLE
```

Comment:
We maybe don't need to care about this anymore, except if there's a reason to not restore the native value of an unavailable sensor?

Reply:
Good question... I think skipping it as now kinda makes sense as it was previously unavailable. Restoring the state would not be correct either?